client/my-sites/plugins/minimum-jetpack-version-notice.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import { warningNotice } from 'state/notices/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+import {
+	isJetpackSite,
+	siteHasMinimumJetpackVersion,
+	getSiteAdminUrl,
+	getSiteDomain
+} from 'state/sites/selectors';
+
+class MinimumJetpackVersionNotice extends Component {
+
+	static propTypes = {
+		translate: PropTypes.func.isRequired,
+		minimumJetpackVersionFailed: PropTypes.bool.isRequired,
+		adminUrl: PropTypes.string,
+		domain: PropTypes.string,
+	};
+
+	showWarning( { minimumJetpackVersionFailed, domain, adminUrl, triggerNotice, translate, siteId } ) {
```

Comment:
Considering the fact that we use Redux middleware for notices, introducing a separate component to trigger a notice is antipattern.

I realize this is the only way to go ahead without refactoring a bunch of legacy plugin code to use Redux, so I'm fine with this approach as a temporary solution. 

So, let's just add a `TODO` comment that we should come back and refactor this one, maybe something along the lines of:

```
// TODO: Refactor to use Redux notices middleware after plugins have been migrated to use Redux
```